aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -75,8 +71,9 @@ const allData = [
 // Define which attributes should be made available for the user
 // to filter upon
 const filtersConfig = [
-  { property: 'role', label: 'Role', filterType: 'CheckBoxGroup' },
+  { property: 'name', label: 'Name', filterType: 'CheckBoxGroup' },
```

Comment:
With this many filters, we probably want to present the filters in a side layer instead of a drop. Similar to: https://deploy-preview-3116--keen-mayer-a86c8b.netlify.app/components/layer/side-drawer-layer?q=layer#filtering

Reply:
Sounds good! The configure columns button was a drop down so I just went along with that, it did feel a bit weird seeing a huge drop down.

Reply:
Would it be a good time to add support for `<DataFilters layer />`?

Reply:
I think that would be nice : )

Reply:
That would be super convenient!